Project/SubZeroDLL/pch.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+// add headers that you want to pre-compile here
```

Comment:
You should re-order your `includes` according to the best practices:
```
"Your libraries"

<C++ Standard Libraries>

<Boost Libraries>

<Windows.h>
```

In this manner you would prevent common mistakes and let the compiler to choose the most relevant symbols for you. 
You should do this in all the header files.

https://stackoverflow.com/a/2762596